packages/genesys-spark-components/src/components/beta/gux-label/example.html

```diff
@@ -0,0 +1,27 @@
+<!--
+    GUX-LABEL
+    Props:
+    fieldStatus - indicates if field is required, optional or neither
+    actionIcon - displays clickable icon, limited to question mark or info icon
+
+    Behaviour:
+    actionIcon click emits guxactioniconclick
+
+    Styling:
+    Text is not truncated, can be single or multiple lines.
+
+    Attributes:
+    for - standard label attribute
+    Global attributes
+
+    Warnings:
+    Placing inputs inside labels gives an implicit for to the label. In this case, a warning may be logged if the element not considered appropriate within a label. This could be for anchors and buttons etc.
+
+-->
+
+<gux-form-field-text-like>
+  <input slot="input" type="text" name="t-1" value="Sample text" />
+  <gux-label slot="label" field-status="“required”" action-icon="“info”"
```

Comment:
Do we need a new component for this? We already provide accessibility and field status (required) to labels that are slotted into our form field components. We can add other needed functionality such as action-icon to the form field components if needed. At first glance this doesn't really seem to be adding any extra utility and is adding complexity. Could you provide some more context as to why we need this new `gux-label` component?

Reply:
Apologies, I tried to find the existing component but totally missed it. I'll work with that one